src/reset.css.js

```diff
@@ -64,6 +64,13 @@ export default createGlobalStyle`
     resize: none;
   }
 
+  input,
+  button,
+  textarea {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Helvetica, Arial,
```

Comment:
What about just using `inherit` here? The rest of the site uses that anyway, so we could avoid duplicating the font stack